### PR TITLE
Work for Eric Rosado

### PR DESCRIPTION
**Pull request by dgilleland**
This is a simple pull request sample.

As we're waiting for @ericdrosado to accept the invitation, I'll assign review responsibilities to @dagilleland  and @SomeContributor.

### CODE REVIEW

**Review by SomeContributor — APPROVED** [review]
Looks good to me - I approve this pull request.

File: questions.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Questions on Pull Requests
+
+I've actually been working with GitHub, using it for my classroom, for the past six and a half years. I've been loving it all the way, and I got into it ***because*** of **GitHub Desktop**.
+
+I do have one quetion, however:
```
Comment (dagilleland) [inline questions.md:5]: Hopefully, the spelling mistake of "quetion" will be fixed at some time.

**Review by dagilleland — APPROVED** [review]
Assuming you will fix this in the future, I approve.

**Comment by ericdrosado** [comment]
👋 Hi @dgilleland! Thanks for the invite. I see that you are well versed with GitHub usage. This all looks good to me. 👍

**Pull request merged**